environment-dev.yml

```diff
@@ -3,12 +3,47 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python >=3.8,<3.12
-  - jinja2 <=3.0.3
+  - python >=3.10,<3.11
+  - aiohttp
+  - birdy
+  - cartopy
+  - cf_xarray
+  - clisops
+  - dask
```

Comment:
Wow, a lot of package added !!!  Do we run out of memory while installing all this?  For Finch and Raven, we used to have to use mock to avoid installing too much and run out of memory during installation.

Reply:
In order to run the notebooks, we need everything here. The memory usage seems to be OK for ReadTheDocs at the moment, so I'm counting my blessings. I imagine Jenkins will be fine with these additions.

Reply:
Jenkins use the production Jupyter env as the runtime environment.  It does not rebuild on each run like for ReadTheDocs or Github.